lib/tasks/release-notes.js

```diff
@@ -0,0 +1,27 @@
+module.exports.displayReleaseNotes = async (ctx, task) => {
+    const got = require('got');
+    const response = await got('https://api.github.com/repos/TryGhost/Ghost/releases', {json: true});
+    const relevantNotes = response.body.filter(note => note.tag_name === ctx.version)[0];
+
+    task.title = 'Fetched release notes';
+
+    if (!relevantNotes) {
+        return;
+    }
+
+    ctx.releaseNotesContainWarning = relevantNotes.body.includes('⚠️');
+    task.output = '\n' + relevantNotes.body + '\n';
```

Comment:
you shouldn't need to do this & modify the task renderer, you should be able to just log things via the UI log method and it'll output correctly

Reply:
Good point - fixed in temporary new commit